libdevcore/SnappyCompress.h

```diff
@@ -22,7 +22,7 @@
  */
 #pragma once
 #include "Common.h"
-#include "easylog.h"
+#include "Log.h"

```

Comment:
This can be removed, too. since `Log.h` has been included in `libdevcore/Common.h`